Formula/genkfs.rb

```diff
@@ -0,0 +1,42 @@
+class Genkfs < Formula
+  desc "Writes a KFS filesystem into a ROM file"
+  homepage "https://github.com/KnightOS/genkfs"
+  url "https://github.com/KnightOS/genkfs/archive/1.2.2.tar.gz"
+
+  depends_on "asciidoc" => :build
+  depends_on "cmake" => :build
+
+  patch :DATA
+
+  def install
+    system "cmake", ".", *std_cmake_args, "-DCMAKE_INSTALL_MANDIR:PATH=#{man}"
+    system "make", "install"
+  end
+
+  test do
+    system bin/"genkfs", "--help"
+    system "man", "genkfs"
+    system "wget", "https://github.com/KnightOS/kernel/releases/download/0.6.11/kernel-TI83p.rom"
```

Comment:
I assume there's a better way to do this...? (jenkins is failing because wget was not added as a build dependency)